packages/celotool/src/cmds/deploy/destroy/oracle.ts

```diff
@@ -1,20 +1,33 @@
 import { addContextMiddleware, ContextArgv, switchToContextCluster } from 'src/lib/context-utils'
+import { CurrencyPair } from 'src/lib/k8s-oracle/base'
 import { getOracleDeployerForContext } from 'src/lib/oracle'
+import yargs from 'yargs'
 import { DestroyArgv } from '../../deploy/destroy'
 
 export const command = 'oracle'
 
 export const describe = 'destroy the oracle package'
 
-type OracleDestroyArgv = DestroyArgv & ContextArgv
+type OracleDestroyArgv = DestroyArgv &
+  ContextArgv & {
+    currencyPair: CurrencyPair
+  }
 
-export const builder = addContextMiddleware
+export const builder = (argv: yargs.Argv) => {
+  return addContextMiddleware(argv).option('currencyPair', {
+    choices: ['CELOUSD', 'CELOEUR', 'CELOBTC'],
```

Comment:
instead of having to change the choices for each initial/upgrade/destroy file, wdyt of putting this .option in a helper function similar to how `addContextMiddleware` works?